apps/condo/domains/contact/schema/Contact.js

```diff
@@ -169,7 +172,10 @@ const Contact = new GQLListSchema('Contact', {
             },
         ],
     },
-    plugins: [uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical(), webHooked()],
+    plugins: [
+        uuided(), versioned(), tracked(), softDeleted(), dvAndSender(), historical(), webHooked(),
+        serviceUserAccessForB2BApp({ schemaConfig: SERVICE_USER_ACCESS_FOR_B2B_APP_CONFIG.Contact }),
```

Comment:
That's syntax is kinda strange.
1. Why do we need to pass config to every singe plugin call?
2. `lowerCamelCase: CAPS_CASE.UpperCamelCase` 🤣

Reply:
1) Config is optional. You can leave it empty, then default config values will be used
2) What's wrong with him?) `propKey: GLOBAL_VARIABLE.SchemaName`. I can move it in a variable with a different name and use it, but does that make sense?